app/controllers/sent_scheduled_messages_controller.rb

```diff
@@ -1,5 +1,9 @@
 class SentScheduledMessagesController < ApplicationController
   def index
-    @sent_scheduled_messages = SentScheduledMessage.order(sent_on: :desc).page(params[:page])
+  	if !params[:slack_username].blank? || !params[:message_body].blank? || !params[:sent_on].blank?
```

Comment:
I find that `thing.present?` is easier to read than `!thing.blank?` and they are logically equivalent. what do you think?